core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1577,12 +1589,18 @@ class ReplicaManager(val config: KafkaConfig,
                 responseMap.put(topicPartition, error)
               }
             }
+
+            breakdown("validateTopicIdInfo") += time.milliseconds() - validateTopicIdInfoStart
```

Comment:
Wouldn't it be the same if you change this line to `breakdown("validateTopicIdInfo") += intervalMarker.markTimeAndReturnLatestInterval(time.milliseconds())` and remove line 1534?

Reply:
Fixed.  Good catch